runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -85,19 +85,23 @@ J9::X86::CodeGenerator::CodeGenerator() :
    cg->setSupportsPartialInlineOfMethodHooks();
    cg->setSupportsInliningOfTypeCoersionMethods();
    cg->setSupportsNewInstanceImplOpt();
-   if (cg->getX86ProcessorInfo().supportsSSE4_1() &&
+   
+   TR_ASSERT(cg->comp()->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE4_1) == cg->getX86ProcessorInfo().supportsSSE4_1(), "supportsSSE4_1() failed\n");
+   TR_ASSERT(cg->comp()->target().cpu.supportsFeature(OMR_FEATURE_X86_SSSE3) == cg->getX86ProcessorInfo().supportsSSSE3(), "supportsSSSE3() failed\n");
+   
+   if (cg->comp()->target().cpu.supportsFeature(OMR_FEATURE_X86_SSE4_1) &&
```

Comment:
I see that we have `comp` already available, so `cg->comp()` could be replaced by `comp`